### PR TITLE
Add new params to get applications endpoint and new properties to application body

### PR DESCRIPTION
**Pull request by ViniciusContini**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -981,6 +981,51 @@ paths:
...
+          description: unique farm identification
+          required: false
+          schema:
+            type: string
+            format: uuid
+            nullable: false
+            example: 2024-06-01T22:52:44.903Z
+        - name: finishedIn
+          in: query
+          description: unique farm identification
```
Comment (throberto) [inline openapi.yaml]: `finishedIn` = `unique farm identification` ?

File: openapi.yaml [hunk 2]
```diff
@@ -981,6 +981,51 @@ paths:
...
+          required: false
+          schema:
+            type: string
+            format: uuid
+            nullable: false
+            example: 075773c3-0f4b-4ac0-b74b-6e777ec0a4bd
+        - name: startedIn
+          in: query
+          description: unique farm identification
```
Comment (throberto) [inline openapi.yaml]: `startedIn` = `unique farm identification` ?

**Review by throberto — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (ViniciusContini) [reply openapi.yaml]: Ajustado.

File: openapi.yaml (hunk 2 above)
Comment (ViniciusContini) [reply openapi.yaml]: Ajustado.

Comment (ViniciusContini) [reply openapi.yaml]: Ajustado.

**Review by EvandroCaslini — APPROVED** [review]

**Pull request merged**